docs/examples.rst

```diff
@@ -277,8 +277,17 @@ select.
 Now start pyffd:
 
 .. code-block:: bash
-
-  # pyffd -f --loglevel=DEBUG -p /var/run/pyffd.pid mdx.fd
+  
+  # CACHE=-C
```

Comment:
-C means turn off cache - maybe be clear about that in the text
I like that you're using env variables to enhance readability!